Resources/Prototypes/Reagents/Consumable/Drink/alcohol.yml

```
@@ -163,8 +163,19 @@
   metabolisms:
     Alcohol:
       effects:
+      #Keeps characters without alcohol tolerance traits at the normal threshold for alcohol poisoning
+      #Harmony start
       - !type:HealthChange
         conditions:
+        - !type:HasTag
+          tag: HeavyweightDrunk
+          invert: true
+        - !type:HasTag
+          tag: LightweightDrunk
+          invert: true
+        - !type:HasTag
+          tag: VeryLightweightDrunk
+          invert: true
```

```suggestion
      - !type:HealthChange
        conditions:
        #Harmony Change start - [Describe effect here?]
        - !type:HasTag
          tag: HeavyweightDrunk
          invert: true
        - !type:HasTag
          tag: LightweightDrunk
          invert: true
        - !type:HasTag
          tag: VeryLightweightDrunk
          invert: true
        # Harmony Change End
```
Don't include untouched upstream code in a Harmony Change comment block & please close your harmony change comment blocks.

Also, optional(?), but it might be nice to describe the effect or reasoning for the added code